### ISSUE TITLE
Don't report empty elements with [aria-hidden="true"]

### ISSUE THREAD

**Issue by bartveneman** [ffoodd/a11y.css#138]
When using empty elements for icons, like with font awesome for example, these elements are still reported when the `aria-hidden` attribute with value `true` is present. Would it be an idea to turn this one off, because the author of the document explicitly states that this element is for styling purposes?

**Comment by ffoodd** [comment]
This test case only means using `aria-hidden` should be double-checked. I think this is useful as a reminder, since some users may not be aware about the whole `aria-hidden` attribute.

However I know some criterias may be annoying. Accessibility experts will find most test cases useless. That's why you can disable specific tests when using Sass, [as mentionned in the README](https://github.com/ffoodd/a11y.css#disabling-tests).

In case you're not a Sass user, you can still use the bookmarklet generator to avoid the advice level. This will add every tests but the 19 advices.

Did I successfully answer to your question?

**Comment by bartveneman** [comment]
I was interested in your opinion on this idea. I understand that these issues should be double checked. Maybe an advice would be better for this particular use case than a warning, but that might be a matter of preference.

I'll close this issue, because my goal (at least discuss it) has been reached.

**Comment by ffoodd** [comment]
Thanks!

I'm still wondering: did you saw this message as a warning (yellow) ? It should appear as an advice ([documented here](http://ffoodd.github.io/a11y.css/advices.html#hidden)) so you may have met a bug :/

**Comment by ffoodd** [comment]
I finally understood that the issue was on the *:empty test that also matches hidden éléments !
I will fix this and let the advice test about using aria-hidden do its job.

Thanks for your comment, and sorry it took me so long to understand !

**Comment by bartveneman** [comment]
Ha, very good! I saw it as a warning indeed.

**Comment by bartveneman** [comment]
Maybe I should add my use case, to make it more clear:

````html
<i class="icon icon--foo" aria-hidden="true"></i><!-- Please do not report a warning here, I've seen it and marked it as hidden -->
<span>Some text</span>
````

**Comment by ffoodd** [comment]
Thank you

**Comment by ffoodd** [comment]
Thank you :)
